pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -22,7 +22,8 @@
     "test": "jest --watch",
     "start": "npm-run-all -p build:watch tsc:watch",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "bump": "npm version patch --no-git-tag-version"
```

Comment:
Any reason to use `npm` here vs `yarn  version --patch --no-git-tag-version`?

Reply:
no real reason, I guess because `yarn` is an additional dependency (to node) I have the habit of reaching for `npm` first. As we use `yarn` as a standard I can change it.